apps/artemis-web/middleware/limiter.ts

```diff
@@ -0,0 +1,57 @@
+import rateLimit from "express-rate-limit";
+require('dotenv').config();
+
+
+function calculateNextResetTime(windowMs) {
+    const d = new Date();
+    d.setMilliseconds(d.getMilliseconds() + windowMs);
+    return d;
+}
+
+function MemoryStore(windowMs, file) {
+    let hits = {};
+    let resetTime = calculateNextResetTime(windowMs);
+
+    this.incr = function (key, cb) {
```

Comment:
are this called by the library package? Or should they be called manually in our code?